events.js

```diff
@@ -1,472 +1,488 @@
 'use strict'
 
 module.exports = function (opts) {
+  var exports = {}
```

Comment:
having the global `module.exports` and this variable is confusing. This var is the thing that get's returned by this func, right? Maybe `eventSomething`..? Processor or Actor.

side node; we have this un-named init thing for this sub-module and more exported funcs with names (clock, connect, etc.)?... :cold_sweat:

Reply:
Agree the opts is quite confusing. One of the other changes I have queued up in removing v2. And when we do this we don't really need this opts.